### PR TITLE
New AKS Blog - Initial commit

### PR DESCRIPTION
**Pull request by chzbrgr71**
This PR has the initial files needed to generate the AKS Engineering Blog. It includes:

- Blog folder with all of the Jekyll files and initial design
- Sample posts
- GitHub Action workflow to generate site

Once we've merged this PR, we can configure the build and add the initial post

### CODE REVIEW

**Review by qpetraroia — APPROVED** [review]

File: blog/_posts/2024-04-01-placeholder-post.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: "New AKS Engineering Blog"
+description: "The new blog is ready!"
+date: 2024-04-01
+author: Brian Redmond
+categories: general
+---
+
+## Welcome to the new AKS Engineering Blog
+
+Full description here
```
Comment (chasewilson) [inline blog/_posts/2024-04-01-placeholder-post.md:11]: Are we supposed to have content here? At least an introduction? Or will this be hidden for now?

File: blog/_posts/2050-08-24-aks-past-present-future.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: "AKS - How we got here"
+description: "An exploration of the past, present, future for Azure Kubernetes Service (AKS)."
+date: 2050-08-24
+author: Jorge Palma
+categories: general
+---
+
+## Introduction
```
Comment (chasewilson) [inline blog/_posts/2050-08-24-aks-past-present-future.md:9]: Same for this one? Are we going to publish it without being full?

**Review by chasewilson — APPROVED** [review]
Not blocking comments as long as the content will either be hidden or completed before publish?

File: blog/_posts/2024-04-01-placeholder-post.md (hunk 1 above)
Comment (chzbrgr71) [reply blog/_posts/2024-04-01-placeholder-post.md:11]: These are just placeholders and will be removed when we add the first post. We need something there to render the site and test

File: blog/_posts/2050-08-24-aks-past-present-future.md (hunk 2 above)
Comment (chzbrgr71) [reply blog/_posts/2050-08-24-aks-past-present-future.md:9]: These are just placeholders and will be removed when we add the first post. We need something there to render the site and test

**Pull request merged**